File: daggercore/src/main/java/com/worldsnas/daggercore/navigator/changehandler/ArcFadeMoveChangeHandlerCompat.java
package com.worldsnas.daggercore.navigator.changehandler;

import com.bluelinelabs.conductor.changehandler.FadeChangeHandler;
import com.bluelinelabs.conductor.changehandler.TransitionChangeHandlerCompat;

public class ArcFadeMoveChangeHandlerCompat extends TransitionChangeHandlerCompat {

    public ArcFadeMoveChangeHandlerCompat() {
        super();
    }

    public ArcFadeMoveChangeHandlerCompat(String... transitionNames) {
        super(new ArcFadeMoveChangeHandler(transitionNames), new FadeChangeHandler());
    }

}

File: navigation/src/androidMain/kotlin/com/worldsnas/navigation/changehandler/ArcFadeMoveChangeHandlerCompat.kt
package com.worldsnas.navigation.changehandler

import androidx.annotation.Keep
import com.bluelinelabs.conductor.changehandler.FadeChangeHandler
import com.bluelinelabs.conductor.changehandler.TransitionChangeHandlerCompat

class ArcFadeMoveChangeHandlerCompat : TransitionChangeHandlerCompat {
    @Keep
    constructor() : super() {
    }

    constructor(vararg transitionNames: String) : super(
        ArcFadeMoveChangeHandler(*transitionNames),
        FadeChangeHandler()
    )
}